zipkin-storage/cassandra/src/main/resources/cassandra-schema.cql

```diff
@@ -0,0 +1,82 @@
+CREATE KEYSPACE IF NOT EXISTS zipkin WITH replication = {'class': 'SimpleStrategy', 'replication_factor': '1'};
+
+CREATE TYPE IF NOT EXISTS zipkin.endpoint (
+    service_name text,
+    ipv4         inet,
+    ipv6         inet,
+    port         smallint,
+);
+
+CREATE TYPE IF NOT EXISTS zipkin.annotation (
+    ts bigint,
+    v  text,
+    ep frozen<endpoint>
+);
+
+CREATE TYPE IF NOT EXISTS zipkin.binary_annotation (
+    k  text,
+    v  blob,
+    t  text,
+    ep frozen<endpoint>
+);
+
+CREATE TABLE IF NOT EXISTS zipkin.traces (
+    trace_id            varint,
+    ts                  timeuuid,
```

Comment:
I'd probably keep this separate from timestamp, since timestamp is nullable when a raw span. ex. ts_id = epoch millis and "ts" = normal timestamp. PS why "ts" not timestamp here an other places for the timestamp field of the span or annotation?